test_data_dont_fit_in_mdblob random failure

Found in the `Test_tribler_main` Jenkins job (build [313](https://jenkins-ci.tribler.org/job/Test_tribler_main/313/console)):

```python
_________________________ test_data_dont_fit_in_mdblob _________________________
[gw0] linux -- Python 3.8.5 /usr/bin/python3

metadata_store = <tribler_core.modules.metadata_store.store.MetadataStore object at 0x7f80a0481b20>

    @pytest.mark.timeout(0)
    @db_session
    def test_data_dont_fit_in_mdblob(metadata_store):
        import random as rng  # pylint: disable=import-outside-toplevel
    
        rng.seed(123)
        md_list = [
            metadata_store.TorrentMetadata(
                title='test' + str(x),
                infohash=random_infohash(rng),
                id_=rng.randint(0, 100000000),
                timestamp=rng.randint(0, 100000000),
            )
            for x in range(0, 1)
        ]
        chunk, index = entries_to_chunk(md_list, chunk_size=1)
        assert index == 1
>       assert len(chunk) == 206
E       assert 205 == 206
E         +205
E         -206

src/tribler-core/tribler_core/modules/metadata_store/tests/test_channel_metadata.py:525: AssertionError
```

--- a/src/tribler-core/tribler_core/modules/metadata_store/tests/test_channel_metadata.py
+++ b/src/tribler-core/tribler_core/modules/metadata_store/tests/test_channel_metadata.py
@@ -16,7 +16,6 @@ import pytest
 
 from tribler_common.simpledefs import CHANNEL_STATE
 
-from tribler_core.exceptions import DuplicateTorrentFileError
 from tribler_core.modules.libtorrent.torrentdef import TorrentDef
 from tribler_core.modules.metadata_store.orm_bindings.channel_metadata import (
     CHANNEL_DIR_NAME_LENGTH,
@@ -24,7 +23,12 @@ from tribler_core.modules.metadata_store.orm_bindings.channel_metadata import (
     entries_to_chunk,
 )
 from tribler_core.modules.metadata_store.orm_bindings.channel_node import COMMITTED, NEW, TODELETE, UPDATED
-from tribler_core.modules.metadata_store.serialization import CHANNEL_TORRENT, COLLECTION_NODE, REGULAR_TORRENT
+from tribler_core.modules.metadata_store.serialization import (
+    CHANNEL_TORRENT,
+    COLLECTION_NODE,
+    REGULAR_TORRENT,
+    int2time,
+)
 from tribler_core.modules.metadata_store.store import HealthItemsPayload
 from tribler_core.tests.tools.common import TESTS_DATA_DIR, TORRENT_UBUNTU_FILE
 from tribler_core.utilities.random_utils import random_infohash
@@ -518,13 +522,14 @@ def test_data_dont_fit_in_mdblob(metadata_store):
             title='test' + str(x),
             infohash=random_infohash(rng),
             id_=rng.randint(0, 100000000),
+            torrent_date=int2time(rng.randint(0, 4000000)),
             timestamp=rng.randint(0, 100000000),
         )
         for x in range(0, 1)
     ]
     chunk, index = entries_to_chunk(md_list, chunk_size=1)
     assert index == 1
-    assert len(chunk) == 206
+    assert len(chunk) == 205
 
     # Test corner case of empty list and/or too big index
     with pytest.raises(Exception):

--- a/src/tribler-core/tribler_core/modules/metadata_store/tests/test_store.py
+++ b/src/tribler-core/tribler_core/modules/metadata_store/tests/test_store.py
@@ -21,6 +21,7 @@ from tribler_core.modules.metadata_store.serialization import (
     DeletedMetadataPayload,
     SignedPayload,
     UnknownBlobTypeException,
+    int2time,
 )
 from tribler_core.modules.metadata_store.tests.test_channel_download import CHANNEL_METADATA_UPDATED
 from tribler_core.tests.tools.common import TESTS_DATA_DIR
@@ -115,12 +116,14 @@ def test_squash_mdblobs(metadata_store):
 
 @db_session
 def test_squash_mdblobs_multiple_chunks(metadata_store):
-    r = random.Random(123)
+    rng = random.Random(123)
     md_list = [
         metadata_store.TorrentMetadata(
-            title=''.join(r.choice(string.ascii_uppercase + string.digits) for _ in range(20)),
-            infohash=random_infohash(),
-            torrent_date=datetime.utcfromtimestamp(100),
+            title=''.join(rng.choice(string.ascii_uppercase + string.digits) for _ in range(20)),
+            infohash=random_infohash(rng),
+            id_=rng.randint(0, 100000000),
+            torrent_date=int2time(rng.randint(0, 4000000)),
+            timestamp=rng.randint(0, 100000000),
         )
         for _ in range(0, 10)
     ]

